app/Main.hs

```diff
@@ -11,17 +11,24 @@ module Main where
 
 import           Universum
 
-import           Language.Haskell.Exts  (Module (..), fromParseResult, parseFileContents,
+import           Language.Haskell.Exts  (Extension, Module (..), fromParseResult,
+                                         parseExtension, parseFileContentsWithExts,
                                          prettyPrint)
 import           Language.Haskell.Names (annotate, loadBase)
-import           System.Directory       (createDirectory, doesDirectoryExist)
+import           Prelude                (id)
```

Comment:
`Universum` has `identity` function, so you don't need to export `id` from `Prelude`